src/amo/reducers/highlyRated.js

```diff
@@ -0,0 +1,33 @@
+import {
+  HIGHLY_RATED_GET,
+  HIGHLY_RATED_LOADED,
+  HIGHLY_RATED_FAILED,
+} from 'core/constants';
+
+const initialState = {
+  count: 0,
+  filters: {},
```

Comment:
why do you need to track `filters` in state for `highlyRated` and `popular` add-on data? It looks like (in your other patch) that the filters are only used to make the API call. The API call populates the data in state so why do you need to save the filters? You should already have the filtered results.